lib/rbs/cli.rb

```diff
@@ -750,5 +751,55 @@ def parse_type_name(string)
         TypeName.new(name: last, namespace: namespace.parent)
       end
     end
+
+    def test_opt options
+      opt_string = options.dirs.map { |dir| "-I #{dir}"}.concat(options.libs.map { |lib| "-r{lib}"}).join(' ')
```

Comment:
Escape the options before `join`. `Shellescape.join` would be the best method for our purpose!